data/mods/Aftershock/items/afs_furniture.json

```diff
@@ -3,24 +3,17 @@
     "type": "furniture",
     "id": "f_atomic_butter_churn",
     "name": "atomic butter churn on a stand",
-    "description": "The Rivtech Atomic Butter Churn in deployed state. It hums menacingly. Ready to add raw milk and salt. It will churn turn the earth is a cinder.",
+    "description": "The Rivtech Atomic Butter Churn in deployed state. It hums menacingly. Ready to add raw milk and salt. It will churn til the earth is a cinder.",
```

Comment:
```suggestion
    "description": "The Rivtech Atomic Butter Churn in deployed state. It hums menacingly. Ready to add raw milk and salt. It will churn till the earth is a cinder.",
```